api/WheelEvent.json

```diff
@@ -344,6 +344,151 @@
             "deprecated": false
           }
         }
+      },
+      "wheelDelta": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "12",
+              "version_removed": "79"
+            },
+            "firefox": {
+              "version_added": "90"
```

Comment:
The reason this is being added to Firefox is that it's some other engine and there was a web compat problem, yet all the other data is false. I'm suspicious 🤨

Reply:
@foolip You'd be right to be suspicious :-). Everything about this starting from the original bug fix feels a bit dodgy. 

As I said, I copied the `MouseWheelEvent` properties across and then added support for FF90. My assumptions here are:
- BCD should match mousewheel 
- We don't expect or want anyone to use these on any browser. So given they are deprecated, marking them as false seems like not such a "bad thing" to be wrong on.

In other words, this is a hack, but the effort of proving the versions seems like a waste of time.

What would you like me to do?

Reply:
@ddbeck (& @foolip )

1. Removed the `mdn_url`s. Not going to create pages for these deprecated properties, so no point linking to something that does not exist.
2. I have matched these to version you suggested (based on chrome version 43. Did not mirror support for Safari on iOS as top level says "false".

Now I appreciate that there are likely issues with v43 matching, but at least we know that users will not expect earlier versions, and since we don't want people to use these anyway, is that good enough?

If not, then what versions should we use? You might reasonably expect that since these are legacy properties they would match the original introduction of the API. So Chrome 31, Webview after 37, Safari after 6.1.

Reply:
> I copied the `MouseWheelEvent` properties across

This does not seems like the right starting point to me. The `WheelEvent` and `MouseWheelEvent` events correspond to the "wheel" and "mousewheel" events, and which of those is supported has been a long-standing compat issue.

Confirming the versions with https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/WheelEvent should be easy once https://github.com/foolip/mdn-bcd-collector/pull/1247 is merged.

Reply:
OK, so with http://staging-dot-mdn-bcd-collector.appspot.com/tests/api/WheelEvent I can confirm that support of `WheelEvent` itself and these added properties goes back to at least Chrome 16, the earliest available on BrowserStack. It's probably older than that, maybe Chrome 1.

I would suggest setting everything except Edge, IE and Firefox to true, so that I and @vinyldarkscratch will come back to it as part of https://github.com/mdn/browser-compat-data/issues/6369. This will require testing on older versions of Chrome and Safari than I have.

Reply:
@foolip Thanks for the advice. Done! Ready for re-review.